modules/apps/depot/depot-test/src/testFunctional/tests/DepotDocuments.testcase

```diff
@@ -138,6 +138,56 @@ definition {
 		DMDocument.viewDefaultCP();
 	}
 
+	@description = "This test makes sure that the Google Drive Shortcut can be added in a depot."
+	@priority = "4"
+	test EditAndDeleteGoogleDoc {
```

Comment:
@Hanlf I'll push this in for now. Can you move all our Google Docs test cases to a single file in a separate ticket so it's easier to maintain? Release prop should be set to false.

Reply:
Ok, I will do it under [LRQA-65541](https://issues.liferay.com/browse/LRQA-65541).